templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -2,9 +2,12 @@
 
 {% block content %}
   <h1>Delete manufacturer?</h1>
-  <form action="" method="post">
-    {% csrf_token %}
+  <div class="btn-group" role="group">
+    <form action="" method="post">
+      {% csrf_token %}
 
-    <input type="submit" value="Yes" class="btn btn-danger">
-  </form>
-{% endblock %}
\ No newline at end of file
+      <input type="submit" value="Yes" class="btn btn-danger">
+    </form>
+    <a href="{% url 'taxi:manufacturer-list' %}" class="btn badge-secondary">Cancel</a>
```

Comment:
The cancel button uses the class `badge-secondary`, which might not be consistent with the styling of other buttons in your application. Consider using `btn-secondary` instead to maintain consistent button styling across your application.